### ISSUE TITLE
rake is a dependency, not only a development dependency

### ISSUE THREAD

**Issue by jamesmeador** [sass/sassc-ruby#60]
I cannot "gem install sassc" without Rake installed:

```
$ gem install sassc -v 1.11.0
Fetching: sassc-1.11.0.gem (100%)
Building native extensions.  This could take a while...
ERROR:  Error installing sassc:
	ERROR: Failed to build gem native extension.

    rake RUBYARCHDIR=/home/spitball/.gem/ruby/gems/sassc-1.11.0/lib RUBYLIBDIR=/home/spitball/.gem/ruby/gems/sassc-1.11.0/lib
sh: rake: command not found

Gem files will remain installed in /home/spitball/.gem/ruby/gems/sassc-1.11.0 for inspection.
Results logged to /home/spitball/.gem/ruby/gems/sassc-1.11.0/ext/gem_make.out
```

If rake is required for installation, I'm pretty sure it should be listed as a real dependency.

**Comment by jamesmeador** [comment]
Looks like this is similar to #42

**Comment by bolandrm** [comment]
Resolved in #63
